sky/engine/core/painting/PictureRecorder.cpp

```diff
@@ -34,21 +34,12 @@ PassRefPtr<Picture> PictureRecorder::endRecording()
     RefPtr<Picture> picture = Picture::create(
         adoptRef(m_pictureRecorder.endRecording()));
     m_canvas->clearSkCanvas();
+    m_canvas->ClearDartWrapper();
     m_canvas = nullptr;
+    ClearDartWrapper();
     return picture.release();
 }
 
-PassRefPtr<Drawable> PictureRecorder::endRecordingAsDrawable()
```

Comment:
That seems pretty speculative to me.  I'm happy to leave this code here, but I'm not sure there's enough support in the compositor do much useful with drawables.